php/7.3-alpine/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM alpine:3.10
+# Por defecto se utiliza la timezone de Buenos Aires
+ENV TZ=America/Argentina/Buenos_Aires
+
+RUN apk --no-cache add \
+    bash \
+    tzdata \
+    php7 \
+    php7-ctype \
+    php7-curl \
+    php7-dom \
+    php7-exif \
+    php7-fileinfo \
+    php7-gd \
+    php7-gmp \
+    php7-iconv \
+    php7-json \
+    php7-mbstring \
+    php7-opcache \
+    php7-openssl \
+    php7-pcntl \
+    php7-pdo_pgsql \
+    php7-phar \
+    php7-session \
+    php7-simplexml \
+    php7-sodium \
+    php7-tokenizer \
+    php7-zip \
+    php7-xml \
+    libsodium \
```

Comment:
Quizá tengas que agergar [php7-sodium](https://pkgs.alpinelinux.org/packages?name=php7-sodium&branch=v3.10). Si Alpine no descarga dependencias, `libsodium` va...

Reply:
[This](https://github.com/SIU-Toba/docker/pull/1/files/12db19c42726e54f050a0346eacf99eaa6101ee4#diff-6ea543cdbf0af7bc4feae23d2c2d75f6R26) ?.